pynumdiff/smooth_finite_difference/_smooth_finite_difference.py

```diff
@@ -66,7 +66,7 @@ def meandiff(x, dt, params=None, options={}, window_size=5, num_iterations=1):
         if 'iterate' in options and options['iterate']:
             num_iterations = params[1]
 
-    kernel = utility._mean_kernel(window_size)
+    kernel = utility.mean_kernel(window_size)
```

Comment:
These are actually called explicitly from other modules, so they're basically public. Changed names to reflect.